VTEX - Delivery Promise Notification API.json

```
@@ -0,0 +1,352 @@
+                                        }
+                                    },
+                                    {
+                                        "availability": 3,
+                                        "deliveryChannel": "pickup-in-point",
+                                        "deliveryZoneIds": ["rio-de-janeiro"],
+                                        "deliveryInfo": {
+                                            "id": "Prime2Day",
+                                            "name": "Same Day",
+                                            "deliveryTime": "1.00:00:00"
+                                        }
+                                    }
+                                ]
+                            }
+                        }
+                    }
+                },
+                "responses": {
+                    "202": {
+                        "description": "Accept",
+                        "content": {
+                            "application/json": {
+                                "schema": {
+                                    "description": "Empty schema."
+                                }
+                            }
+                        }
+                    },
+                    "400": {
+                        "description": "Bad request",
+                        "content": {
+                            "application/json": {
+                                "schema": {
+                                    "type": "object",
+                                    "properties": {
+                                        "error": {
+                                            "type": "string",
+                                            "description": "Error message. Invalid JSON or non-registered items."
+                                        }
+                                    }
+                                },
+                                "example": {
+                                    "error": "Invalid JSON or non-registered items."
+                                }
+                            }
+                        }
+                    },
+                    "401": {
+                        "description": "Unauthorized",
+                        "content": {
+                            "application/json": {
+                                "schema": {
+                                    "type": "object",
+                                    "properties": {
+                                        "error": {
+                                            "type": "string",
+                                            "description": "Error message. Invalid app key/token."
+                                        }
+                                    }
+                                },
+                                "example": {
+                                    "error": "Invalid app key/token."
```

estamos mudando a nomenclatura aqui com a nova experiência de API keys:

```suggestion
                                            "description": "Error message. Invalid API key or token."
                                        }
                                    }
                                },
                                "example": {
                                    "error": "Invalid API key or token."
```